src/SignalR/clients/csharp/Client.Core/src/Microsoft.AspNetCore.SignalR.Client.Core.csproj

```diff
@@ -4,7 +4,6 @@
     <Description>Client for ASP.NET Core SignalR</Description>
     <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0;netstandard2.1</TargetFrameworks>
     <RootNamespace>Microsoft.AspNetCore.SignalR.Client</RootNamespace>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why?

Reply:
This was part of the implementation that used `SkipLocalsInitAttribute`. I have removed it, along with my introduction to `<AllowUnsafeBlocks>`. So the net change is scoped specifically to `src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs`, and the private `PackageStreamingParams` method. I can squash the changes into one commit so that this doesn't show up in the change log if that is helpful.

Reply:
Ah this is removed. I thought it was added 👍🏾